Dockerfile

```diff
@@ -32,23 +55,18 @@ RUN curl -fsSLO "$RCON_URL" \
     && tar xfz rcon-0.10.3-amd64_linux.tar.gz \
     && chmod +x "rcon-0.10.3-amd64_linux/$RCON_BINARY" \
     && mv "rcon-0.10.3-amd64_linux/$RCON_BINARY" "/usr/local/bin/${RCON_BINARY}" \
-    && ln -s "/usr/local/bin/${RCON_BINARY}" /usr/local/bin/rconcli \
     && rm -Rf rcon-0.10.3-amd64_linux rcon-0.10.3-amd64_linux.tar.gz
 
-COPY --chown=steam:steam --chmod=755 backupmanager.sh servermanager.sh includes/* /
-
-EXPOSE 8211/udp
-EXPOSE 25575/tcp
-
-# Workaround for 'Named Volumes'
-RUN mkdir /palworld \
-    && chown steam:steam /palworld
-
-VOLUME [ "/palworld" ]
+COPY --chown=steam:steam --chmod=755 scripts/ /scripts
+COPY --chown=steam:steam --chmod=755 includes/ /includes
+COPY --chown=steam:steam --chmod=755 configs/ /configs
+COPY --chown=steam:steam --chmod=755 entrypoint.sh /
 
-USER steam
+RUN ln -s /scripts/backupmanager.sh /usr/local/bin/backup_manager \
```

Comment:
ln -s /scripts/backupmanager.sh /usr/local/bin/backupmanager

dont like the _ in there, servermanager.sh and backupmanager.sh arent to long and readability is great. Why make the syntax more "alien" 😃

Reply:
Slipped, sorry 😅